.github/workflows/build-and-publish.yml

```diff
@@ -19,3 +24,12 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         SOURCE_DIR: 'generated' 
         DEST_DIR: 'trackerblocking/config/v1'
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
```

Comment:
Does this get cached? Should we have hour here?

Alternatively we could take it from the generated version in the config?